### PR TITLE
Handle julia environments

### PR DESCRIPTION
**Pull request by davidanthoff**
@ZacLN Could you take a look at this? In theory it should complete our handling of julia env changes, but in practice it doesn't yet work. As far as I can tell it handles a non-standard environment well at startup, but I think we still need to do something else when we handle the `"julia/activateenvironment"` notification. But I'm not sure what exactly, I don't fully understand the whole innards of the LS enough to fix that. Any pointers or additional commits would be great!

### CODE REVIEW

**Comment by ZacLN** [comment]
Yup this looks like it should work from a quick look, I'll investigate

**Comment by ZacLN** [comment]
Switch code from l137 of misc.jl for the following and this seems to work a treat!

```julia
for (uri, doc) in server.documents
        parse_all(doc, server)
    end
```

File: src/requests/misc.jl [hunk 1]
```diff
@@ -116,14 +116,28 @@ function process(r::JSONRPC.Request{Val{Symbol("julia/getCurrentBlockOffsetRange
...
-end
\ No newline at end of file
+    server.env_path = r.params
+    server.symbol_server = StaticLint.SymbolServer.SymbolServerProcess(depot = server.depot_path, environment=server.env_path)
+    @info "Restarted symbol server"
+    server.packages = StaticLint.SymbolServer.getstore(server.symbol_server)
+    @info "StaticLint store set"
+    kill(server.symbol_server)
+
+    if server.workspaceFolders != nothing
```
Comment (ZacLN) [inline src/requests/misc.jl]: Switch this for 

```julia
for (uri, doc) in server.documents
        parse_all(doc, server)
    end
```

Comment (davidanthoff) [reply src/requests/misc.jl]: Done, thanks!

**Pull request merged**